source/containers.tex

```diff
@@ -17922,18 +17910,14 @@
 
 \indexlibraryctor{flat_multiset}%
 \begin{itemdecl}
-template<class Allocator>
-  flat_multiset(sorted_equivalent_t s, const container_type& cont, const Allocator& a);
-template<class Allocator>
+template<class Alloc>
+  flat_multiset(sorted_equivalent_t s, const container_type& cont, const Alloc& a);
+template<class Alloc>
   flat_multiset(sorted_equivalent_t s, const container_type& cont,
                 const key_compare& comp, const Allocator& a);
```

Comment:
```suggestion
                const key_compare& comp, const Alloc& a);
```

Reply:
This is the only actual error I found. This needs to be fixed. The missing param name should be fixed IMHO. The other two comments don't require action.